libs/net/dal/Services/ContentService.cs

```diff
@@ -131,6 +131,10 @@ public IPaged<Content> FindWithDatabase(ContentFilter filter, bool asNoTracking
         if (filter.ProductIds?.Any() == true)
             query = query.Where(c => filter.ProductIds.Contains(c.ProductId));
 
+        // TODO
+        if(filter.SourceIds?.Any() == true)
+            query = query.Where(c => filter.SourceIds.Contains(c.SourceId ?? 0));
```

Comment:
Threw an error without the ?? 0, is this ok? Will remove TODO after resovled

Reply:
I think this will result in a subpar SQL statement.  I think you can just `c.SourceId!`, which essentially tells it to ignore nullable for this scenario.

Reply:
Hmm I still get the error for some reason, which doesn't make sense

Reply:
Had to cast it to an int first..